package-support.json

```diff
@@ -0,0 +1,16 @@
+{
+  "versions": [
+    {
+      "version": "*",
+      "target": {
+        "node": "lts"
+      },
+      "response": {
+        "type": "regular-7"
```

Comment:
We should probably check with @nainaz in terms of what we want to say here. The same answer will likely apply across a number of modules.

Reply:
@mhdawson yes that makes sense.

Reply:
Just checked with @nainaz and using regular-7 is good,  so  @helio-frota  i think we are good to merge these and the others